msvc/mono.def

```diff
@@ -1,6 +1,4 @@
 ; file generated by create-windef.pl
-EXPORTS
```

Comment:
The intention was to remove the library name since it can be different based on build configuration but the EXPORT keyword should still be around or the .def file will be incorrect. So the fix should be to just have the line:

print OUT "EXPORTS\n";

in the script.